### PR TITLE
Apply some more linting rules

### PR DESCRIPTION
**Pull request by james-d-mitchell**

### CODE REVIEW

File: gap/io.gi [hunk 1]
```diff
@@ -908,24 +909,23 @@ function(func, s)
...
   local D;
   D := MakeImmutable(DigraphFromSparse6String(IsMutableDigraph, s));
   SetIsSymmetricDigraph(D, true);
   return D;
 end);
 
 InstallMethod(DigraphFromSparse6String, "for a function and a string",
-[IsFunction, IsString],
-DigraphFromSparse6StringCons);
+[IsFunction, IsString], DigraphFromSparse6StringCons);
```
Comment (mtorpey) [inline gap/io.gi:928]: Why?

File: gap/isomorph.gi [hunk 2]
```diff
@@ -292,23 +284,13 @@ function(D)
...
 
 InstallMethod(BlissAutomorphismGroup, "for a digraph and vertex coloring",
-[IsDigraph, IsHomogeneousList],
-function(D, colors)
-  return BLISS_DATA(D,
-                    colors,
-                    fail,
-                    "AutomorphismGroup")[1];
-end);
+[IsDigraph, IsHomogeneousList], {D, colors} -> BLISS_DATA(D, colors, fail)[1]);
```
Comment (mtorpey) [inline gap/isomorph.gi:287]: We definitely don't want the last argument in these?

**Review by mtorpey — APPROVED** [review]

File: gap/io.gi (hunk 1 above)
Comment (james-d-mitchell) [reply gap/io.gi:928]: As I think Reimer Behrens once said: "vertical screen real estate is at a premium", so one less line is a good thing from this perspective.

File: gap/isomorph.gi (hunk 2 above)
Comment (james-d-mitchell) [reply gap/isomorph.gi:287]: Definitely don't, I think this was a remnant of some earlier code.

**Pull request merged**